js/pint-types-bundle/index.ts

```diff
@@ -25,13 +24,20 @@ export const definitions = {
                         Saft: null,
                     },
                 },
+                AssetId: "u32",
```

Comment:
can we somehow use the resources/types.json file? so we only have to maintain the types in one spot?

Reply:
Or a script that can migrate:
-  `js/pint-types-bundle/index.ts` -> `resouces/types.json`
-  `resources/types.json` -> `js/pint-types-bundle/index.ts`
?

Reply:
the benefit of having this directly available would be that a user can copy types.json right away and use the polkadot UI.
But if we eventually roll out this js folder as a standalone folder, we need to separate these anyway.

So I'm in favor of a `js/pint-types-bundle/index.ts -> resouces/types.json` pipeline

Reply:
> the benefit of having this directly available would be that a user can copy types.json right away and use the polkadot UI.

agree, we can embed the script in `/build.rs`